File: MissingNumber/MN.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        if nums[-1]!= len(nums): return len(nums)
        if nums[0]!=0:return 0
        for i in range(0,len(nums)):
            if nums[i]!= i:return i
        return -1

File: MissingNumber/MN.java
class Solution {
    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        if(nums[nums.length-1]!=nums.length){
            return nums.length;
        }
        if(nums[0]!=0) return 0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]-nums[0]!=i) {
               return i; 
            }
        }
        return -1;
    }
}